src/app.js

```diff
@@ -0,0 +1,6 @@
+import Api from './Api';
+
+// eslint-disable-next-line no-unused-vars
+const a = new Api(); // Test
```

Comment:
We were creating this object with `{ url: extwrt.globals.wikicolorUrl }`, but that global config is gone now. Maybe just pass the URL in manually here for now? I'm not sure.

Reply:
Oops, I think this went away in my testing and then the rebase. I'm bringing that back in.